types/src/client.rs

```diff
@@ -17,7 +17,18 @@ pub struct Subscription<Notif> {
 	pub marker: PhantomData<Notif>,
 }
 
+impl<Notif> std::fmt::Debug for Subscription<Notif> {
+	fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+		f.debug_struct("Subscription")
+			.field("to_back", &self.to_back)
+			.field("notifs_rx", &self.notifs_rx)
+			.field("id", &self.id)
+			.finish()
+	}
+}
+
 /// Active NotificationHandler on a Client.
+#[derive(Debug)]
```

Comment:
This one has generic `PhantomData` too. I reckon we should be consistent about either having manual impls for both skipping the marker, or derives on both.